### DISCUSSION TITLE
pages/UA-PNP/writer_resources/writer_resources_doc/reference/using_giscus

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#77] [Q&A]
# pages/UA-PNP/writer_resources/writer_resources_doc/reference/using_giscus

Giscus is an open-source tool that lets users add comments about the topic. All comments are collected in GitHub Discussions. By default, we use HCLSoftware-Documentation-Home repository to receive all the feedback.

https://github01.hclpnp.com/pages/UA-PNP/writer_resources/writer_resources_doc/reference/using_giscus.html

**Comment by Sheejavijayan** [comment]
Test
